### ISSUE TITLE
LDAP Connection fail after user logout

### ISSUE THREAD

**Issue by Kaauw** [ansible/awx#11159]
### Please confirm the following

- [X] I agree to follow this project's [code of conduct](http://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
- [X] I have checked the [current issues](https://github.com/ansible/awx/issues) for duplicates.
- [X] I understand that AWX is open source software provided for free and that I am not entitled to status updates or other assurances.

### Summary

Hello Team,

I've managed to set up ldap authentication on Ansible AWX.

Tested on Tower 17X (Tower Trial) And worked well.

I loaded the 17 Configuration to 19X AWX.

I can connect to AWX Using LDAP account for the first connection. But when i logout and retry to connect, it fail.

### AWX version

19.3.0

### Installation method

kubernetes

### Modifications

yes

### Web browser

Chrome

### Steps to reproduce

1. Set up LDAP Auth
2. Connect with user present on LDAP --> It works
3. Disconnect
4. Try to reconnect using the same account --> It failed

### Expected results

User should be able to reconnect.

### Actual results

Users cannot reconnect (Name/ password incorrect)

### Ldap config
####Main 
URI : ldap://ad.bde.lab:389 --> Not LDAP(s)
GroupType: NesstedActiveDirectoryGroupType
TLS : Disable 

####  LDAP Bind DN
`CN=ansible_tower_svc,OU=Comptes de service,OU=Admin,OU=Contoso,DC=bde,DC=lab`

#### LDAP USER SEARCH

```json
[
  [
    "OU=Technique,OU=INTERDATA,OU=IJNEXT,DC=bde,DC=lab",
    "SCOPE_SUBTREE",
    "(sAMAccountName=%(user)s)"
  ],
  [
    "OU=Technique,OU=TENEDIS,OU=IJNEXT,DC=bde,DC=lab",
    "SCOPE_SUBTREE",
    "(sAMAccountName=%(user)s)"
  ],
  [
    "OU=Users,OU=Admin,OU=IJNEXT,DC=bde,DC=lab",
    "SCOPE_SUBTREE",
    "(sAMAccountName=%(user)s)"
  ]
]
```

#### LDAP Group Search
```json
[
  "DC=bde,DC=lab",
  "SCOPE_SUBTREE",
  "(objectClass=group)"
]
```

#### LDAP User Flags by Group
```json
{
  "is_superuser": [
    "CN=gg_tower_admin,OU=Groupes,OU=Admin,OU=IJNEXT,DC=bde,DC=lab"
  ]
}
```

#### Org mapping 
```json
{
  "Interdata": {
    "remove_admins": false,
    "remove_users": false,
    "users": [
      "CN=gg_itd_technique,OU=Groupes,OU=Admin,OU=IJNEXT,DC=bde,DC=lab"
    ],
    "admins": "CN=gg_itd_tower_admin,OU=Groupes,OU=Admin,OU=IJNEXT,DC=bde,DC=lab"
  },
  "Tenedis": {
    "remove_admins": false,
    "remove_users": false,
    "users": [
      "CN=gg_tenedis_technique,OU=Groupes,OU=Admin,OU=IJNEXT,DC=bde,DC=lab"
    ],
    "admins": "CN=gg_tenedis_tower_admin,OU=Groupes,OU=Admin,OU=IJNEXT,DC=bde,DC=lab"
  }
}

```

#### Team Mapping
```json
{
  "Exploitation": {
    "remove": true,
    "users": "CN=gg_itd_technique_noc,OU=Groupes,OU=Admin,OU=IJNEXT,DC=bde,DC=lab",
    "organization": "Interdata"
  },
  "Ingenierie routage": {
    "remove": true,
    "users": "CN=gg_itd_technique_routage,OU=Groupes,OU=Admin,OU=IJNEXT,DC=bde,DC=lab",
    "organization": "Interdata"
  },
  "Service Informatique": {
    "remove": true,
    "users": "CN=gg_itd_technique_si,OU=Groupes,OU=Admin,OU=IJNEXT,DC=bde,DC=lab",
    "organization": "Interdata"
  },
  "Tenedis Général": {
    "remove": true,
    "users": "CN=gg_tenedis_technique_general,OU=Groupes,OU=Admin,OU=IJNEXT,DC=bde,DC=lab",
    "organization": "Tenedis"
  }
}
```

### Kubectl logs "pods" -c awx-itd-web

```

! First Login -->
[pid: 42|app: 0|req: 140/453] 10.244.2.1 () {54 vars in 2395 bytes} [Tue Sep 28 14:49:06 2021] POST /api/login/ => generated 6011
 bytes in 239 msecs (HTTP/1.1 401) 10 headers in 493 bytes (1 switches on core 0)
10.244.2.1 - - [28/Sep/2021:14:49:06 +0000] "POST /api/login/ HTTP/1.1" 401 6011 "http://172.17.130.128/" "Mozilla/5.0 (Windows N
T 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4577.82 Safari/537.36" "-"
2021-09-28 14:49:09,422 DEBUG    [63794450c1bd4981984958ab148e8069] awx.analytics.performance request: <WSGIRequest: GET '/api/lo
gin/'>, response_time: 0.057s
[pid: 44|app: 0|req: 97/454] 10.244.2.1 () {48 vars in 2244 bytes} [Tue Sep 28 14:49:09 2021] GET /api/login/ => generated 5726 b
ytes in 59 msecs (HTTP/1.1 200) 10 headers in 483 bytes (1 switches on core 0)
10.244.2.1 - - [28/Sep/2021:14:49:09 +0000] "GET /api/login/ HTTP/1.1" 200 5726 "http://172.17.130.128/" "Mozilla/5.0 (Windows NT
 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4577.82 Safari/537.36" "-"
2021-09-28 14:49:09,521 DEBUG    [4445cc2fa2764cda8056cfe4cd601b95] django_auth_ldap search_s('OU=Technique,OU=INTERDATA,OU=IJNEX
T,DC=bde,DC=lab', 2, '(sAMAccountName=%(user)s)') returned 1 objects: cn=bart tabac,ou=routage,ou=technique,ou=interdata,ou=ijnex
t,dc=bde,dc=lab
2021-09-28 14:49:09,521 DEBUG    [4445cc2fa2764cda8056cfe4cd601b95] django_auth_ldap search_s('OU=Technique,OU=TENEDIS,OU=IJNEXT,
DC=bde,DC=lab', 2, '(sAMAccountName=%(user)s)') returned 0 objects:
2021-09-28 14:49:09,521 DEBUG    [4445cc2fa2764cda8056cfe4cd601b95] django_auth_ldap search_s('OU=Users,OU=Admin,OU=IJNEXT,DC=bde
,DC=lab', 2, '(sAMAccountName=%(user)s)') returned 0 objects:
--> Creating User
2021-09-28 14:49:09,525 DEBUG    [4445cc2fa2764cda8056cfe4cd601b95] django_auth_ldap Creating Django user btabac
2021-09-28 14:49:09,525 DEBUG    [4445cc2fa2764cda8056cfe4cd601b95] django_auth_ldap Populating Django user btabac
2021-09-28 14:49:09,525 WARNING  [4445cc2fa2764cda8056cfe4cd601b95] django_auth_ldap cn=bart tabac,ou=routage,ou=technique,ou=int
erdata,ou=ijnext,dc=bde,dc=lab does not have a value for the attribute mail
2021-09-28 14:49:09,528 DEBUG    [4445cc2fa2764cda8056cfe4cd601b95] django_auth_ldap search_s('DC=bde,DC=lab', 2, '(&(objectClass
=group)(|(member=cn=bart tabac,ou=routage,ou=technique,ou=interdata,ou=ijnext,dc=bde,dc=lab)))') returned 3 objects: cn=gg_itd_te
chnique_routage,ou=groupes,ou=admin,ou=ijnext,dc=bde,dc=lab; cn=gg_ijnext,ou=groupes,ou=admin,ou=ijnext,dc=bde,dc=lab; cn=gg_itd_
tower_admin,ou=groupes,ou=admin,ou=ijnext,dc=bde,dc=lab
2021-09-28 14:49:09,529 DEBUG    [4445cc2fa2764cda8056cfe4cd601b95] django_auth_ldap search_s('DC=bde,DC=lab', 2, '(&(objectClass
=group)(|(member=cn=gg_itd_technique_routage,ou=groupes,ou=admin,ou=ijnext,dc=bde,dc=lab)(member=cn=gg_itd_tower_admin,ou=groupes
,ou=admin,ou=ijnext,dc=bde,dc=lab)(member=cn=gg_ijnext,ou=groupes,ou=admin,ou=ijnext,dc=bde,dc=lab)))') returned 1 objects: cn=gg
_itd_technique,ou=groupes,ou=admin,ou=ijnext,dc=bde,dc=lab
2021-09-28 14:49:09,530 DEBUG    [4445cc2fa2764cda8056cfe4cd601b95] django_auth_ldap search_s('DC=bde,DC=lab', 2, '(&(objectClass
=group)(|(member=cn=gg_itd_technique,ou=groupes,ou=admin,ou=ijnext,dc=bde,dc=lab)))') returned 1 objects: cn=gg_itd,ou=groupes,ou
=admin,ou=ijnext,dc=bde,dc=lab
2021-09-28 14:49:09,531 DEBUG    [4445cc2fa2764cda8056cfe4cd601b95] django_auth_ldap search_s('DC=bde,DC=lab', 2, '(&(objectClass
=group)(|(member=cn=gg_itd,ou=groupes,ou=admin,ou=ijnext,dc=bde,dc=lab)))') returned 1 objects: cn=gg_ijnext,ou=groupes,ou=admin,
ou=ijnext,dc=bde,dc=lab
2021-09-28 14:49:09,531 DEBUG    [4445cc2fa2764cda8056cfe4cd601b95] django_auth_ldap cn=bart tabac,ou=routage,ou=technique,ou=int
erdata,ou=ijnext,dc=bde,dc=lab is not a member of cn=gg_tower_admin,ou=groupes,ou=admin,ou=ijnext,dc=bde,dc=lab
2021-09-28 14:49:09,544 DEBUG    [4445cc2fa2764cda8056cfe4cd601b95] django_auth_ldap cn=bart tabac,ou=routage,ou=technique,ou=int
erdata,ou=ijnext,dc=bde,dc=lab is a member of cn=gg_itd_tower_admin,ou=groupes,ou=admin,ou=ijnext,dc=bde,dc=lab
2021-09-28 14:49:09,590 DEBUG    [4445cc2fa2764cda8056cfe4cd601b95] django_auth_ldap cn=bart tabac,ou=routage,ou=technique,ou=int
erdata,ou=ijnext,dc=bde,dc=lab is a member of cn=gg_itd_technique,ou=groupes,ou=admin,ou=ijnext,dc=bde,dc=lab
2021-09-28 14:49:09,621 DEBUG    [4445cc2fa2764cda8056cfe4cd601b95] django_auth_ldap cn=bart tabac,ou=routage,ou=technique,ou=int
erdata,ou=ijnext,dc=bde,dc=lab is not a member of cn=gg_tenedis_tower_admin,ou=groupes,ou=admin,ou=ijnext,dc=bde,dc=lab
2021-09-28 14:49:09,625 DEBUG    [4445cc2fa2764cda8056cfe4cd601b95] django_auth_ldap cn=bart tabac,ou=routage,ou=technique,ou=int
erdata,ou=ijnext,dc=bde,dc=lab is not a member of cn=gg_tenedis_technique,ou=groupes,ou=admin,ou=ijnext,dc=bde,dc=lab
2021-09-28 14:49:09,634 DEBUG    [4445cc2fa2764cda8056cfe4cd601b95] django_auth_ldap cn=bart tabac,ou=routage,ou=technique,ou=int
erdata,ou=ijnext,dc=bde,dc=lab is not a member of cn=gg_itd_technique_noc,ou=groupes,ou=admin,ou=ijnext,dc=bde,dc=lab
2021-09-28 14:49:09,641 DEBUG    [4445cc2fa2764cda8056cfe4cd601b95] django_auth_ldap cn=bart tabac,ou=routage,ou=technique,ou=int
erdata,ou=ijnext,dc=bde,dc=lab is a member of cn=gg_itd_technique_routage,ou=groupes,ou=admin,ou=ijnext,dc=bde,dc=lab
2021-09-28 14:49:09,675 DEBUG    [4445cc2fa2764cda8056cfe4cd601b95] django_auth_ldap cn=bart tabac,ou=routage,ou=technique,ou=int
erdata,ou=ijnext,dc=bde,dc=lab is not a member of cn=gg_itd_technique_si,ou=groupes,ou=admin,ou=ijnext,dc=bde,dc=lab
2021-09-28 14:49:09,682 DEBUG    [4445cc2fa2764cda8056cfe4cd601b95] django_auth_ldap cn=bart tabac,ou=routage,ou=technique,ou=int
erdata,ou=ijnext,dc=bde,dc=lab is not a member of cn=gg_tenedis_technique_general,ou=groupes,ou=admin,ou=ijnext,dc=bde,dc=lab
2021-09-28 14:49:09,705 INFO     [4445cc2fa2764cda8056cfe4cd601b95] awx.api.generics User btabac logged in from 10.244.2.1
2021-09-28 14:49:10,240 DEBUG    [4445cc2fa2764cda8056cfe4cd601b95] awx.analytics.performance request: <WSGIRequest: POST '/api/l
ogin/'>, response_time: 0.799s
[pid: 44|app: 0|req: 98/455] 10.244.2.1 () {54 vars in 2395 bytes} [Tue Sep 28 14:49:09 2021] POST /api/login/ => generated 0 byt
es in 801 msecs (HTTP/1.1 302) 15 headers in 2102 bytes (1 switches on core 0)
10.244.2.1 - - [28/Sep/2021:14:49:10 +0000] "POST /api/login/ HTTP/1.1" 302 0 "http://172.17.130.128/" "Mozilla/5.0 (Windows NT 1
0.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4577.82 Safari/537.36" "-"
2021-09-28 14:49:10,403 DEBUG    [0989e49df29c452b82f8c7b38a53c610] awx.analytics.performance request: <WSGIRequest: GET '/api/v2
/config/'>, response_time: 0.147s
[pid: 43|app: 0|req: 63/456] 10.244.2.1 () {48 vars in 2288 bytes} [Tue Sep 28 14:49:10 2021] GET /api/v2/config/ => generated 22
14 bytes in 149 msecs (HTTP/1.1 200) 14 headers in 589 bytes (1 switches on core 0)
10.244.2.1 - - [28/Sep/2021:14:49:10 +0000] "GET /api/v2/config/ HTTP/1.1" 200 2214 "http://172.17.130.128/" "Mozilla/5.0 (Window
s NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4577.82 Safari/537.36" "-"
2021-09-28 14:49:10,705 DEBUG    [c174d9153d3e4358b6a0d68f3eda5de8] awx.analytics.performance request: <WSGIRequest: GET '/api/v2
/config/'>, response_time: 0.259s
10.244.2.1 - - [28/Sep/2021:14:49:10 +0000] "GET /api/v2/config/ HTTP/1.1" 200 2214 "http://172.17.130.128/" "Mozilla/5.0 (Window
s NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4577.82 Safari/537.36" "-"
[pid: 41|app: 0|req: 84/457] 10.244.2.1 () {48 vars in 2288 bytes} [Tue Sep 28 14:49:10 2021] GET /api/v2/config/ => generated 22
14 bytes in 262 msecs (HTTP/1.1 200) 14 headers in 589 bytes (1 switches on core 0)
2021-09-28 14:49:11,017 DEBUG    [a9ceb3e69bc14d90a8ce8859b35cee2d] awx.analytics.performance request: <WSGIRequest: GET '/api/v2
/users/8/admin_of_organizations/'>, response_time: 0.280s
[pid: 41|app: 0|req: 85/458] 10.244.2.1 () {48 vars in 2336 bytes} [Tue Sep 28 14:49:10 2021] GET /api/v2/users/8/admin_of_organi
zations/ => generated 3418 bytes in 283 msecs (HTTP/1.1 200) 14 headers in 575 bytes (1 switches on core 0)
10.244.2.1 - - [28/Sep/2021:14:49:11 +0000] "GET /api/v2/users/8/admin_of_organizations/ HTTP/1.1" 200 3418 "http://172.17.130.12
8/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4577.82 Safari/537.36" "-"
2021-09-28 14:49:11,297 DEBUG    [739b7ed4314042be93faf8a639eb1448] awx.analytics.performance request: <WSGIRequest: GET '/api/v2
/workflow_approvals/?page_size=1&status=pending'>, response_time: 0.206s
[pid: 43|app: 0|req: 64/459] 10.244.2.1 () {48 vars in 2365 bytes} [Tue Sep 28 14:49:11 2021] GET /api/v2/workflow_approvals/?pag
e_size=1&status=pending => generated 52 bytes in 209 msecs (HTTP/1.1 200) 14 headers in 579 bytes (1 switches on core 0)
10.244.2.1 - - [28/Sep/2021:14:49:11 +0000] "GET /api/v2/workflow_approvals/?page_size=1&status=pending HTTP/1.1" 200 52 "http://
172.17.130.128/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4577.82 Safari/537
.36" "-"
2021-09-28 14:49:12,454 DEBUG    [8630dbbbf46948099c14dbfb7b15120b] awx.analytics.performance request: <WSGIRequest: GET '/api/v2
/config/'>, response_time: 0.162s
[pid: 42|app: 0|req: 141/460] 10.244.2.1 () {48 vars in 2288 bytes} [Tue Sep 28 14:49:12 2021] GET /api/v2/config/ => generated 2
214 bytes in 164 msecs (HTTP/1.1 200) 14 headers in 589 bytes (1 switches on core 0)
10.244.2.1 - - [28/Sep/2021:14:49:12 +0000] "GET /api/v2/config/ HTTP/1.1" 200 2214 "http://172.17.130.128/" "Mozilla/5.0 (Window
s NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4577.82 Safari/537.36" "-"
2021-09-28 14:49:12,553 DEBUG    [af319d53afc943999d148a3925f8c0f7] awx.analytics.performance request: <WSGIRequest: GET '/api/v2
/organizations/?page_size=1&role_level=execution_environment_admin_role'>, response_time: 0.228s
10.244.2.1 - - [28/Sep/2021:14:49:12 +0000] "GET /api/v2/organizations/?page_size=1&role_level=execution_environment_admin_role H
TTP/1.1" 200 3418 "http://172.17.130.128/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chro
me/93.0.4577.82 Safari/537.36" "-"
[pid: 45|app: 0|req: 73/461] 10.244.2.1 () {48 vars in 2414 bytes} [Tue Sep 28 14:49:12 2021] GET /api/v2/organizations/?page_siz
e=1&role_level=execution_environment_admin_role => generated 3418 bytes in 230 msecs (HTTP/1.1 200) 14 headers in 581 bytes (1 sw
itches on core 0)
2021-09-28 14:49:12,889 DEBUG    [5f12c3630f484764821712dfb250a0e3] awx.analytics.performance request: <WSGIRequest: GET '/api/v2
/users/8/admin_of_organizations/'>, response_time: 0.296s
[pid: 42|app: 0|req: 142/462] 10.244.2.1 () {48 vars in 2337 bytes} [Tue Sep 28 14:49:12 2021] GET /api/v2/users/8/admin_of_organ
izations/ => generated 3418 bytes in 297 msecs (HTTP/1.1 200) 14 headers in 575 bytes (1 switches on core 0)
10.244.2.1 - - [28/Sep/2021:14:49:12 +0000] "GET /api/v2/users/8/admin_of_organizations/ HTTP/1.1" 200 3418 "http://172.17.130.12
8/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4577.82 Safari/537.36" "-"
2021-09-28 14:49:12,939 DEBUG    [201f9cc171ab4efe8bdf52554f9dff98] awx.analytics.performance request: <WSGIRequest: GET '/api/v2
/dashboard/graphs/jobs/?job_type=all&period=month'>, response_time: 0.256s
[pid: 41|app: 0|req: 86/463] 10.244.2.1 () {48 vars in 2369 bytes} [Tue Sep 28 14:49:12 2021] GET /api/v2/dashboard/graphs/jobs/?
job_type=all&period=month => generated 1124 bytes in 258 msecs (HTTP/1.1 200) 14 headers in 575 bytes (1 switches on core 0)
10.244.2.1 - - [28/Sep/2021:14:49:12 +0000] "GET /api/v2/dashboard/graphs/jobs/?job_type=all&period=month HTTP/1.1" 200 1124 "htt
p://172.17.130.128/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4577.82 Safari
/537.36" "-"
2021-09-28 14:49:12,984 DEBUG    [52594a0c2bae4d1e9a23bb12095f2a2c] awx.analytics.performance request: <WSGIRequest: GET '/api/v2
/dashboard/'>, response_time: 0.290s
10.244.2.1 - - [28/Sep/2021:14:49:12 +0000] "GET /api/v2/dashboard/ HTTP/1.1" 200 1428 "http://172.17.130.128/" "Mozilla/5.0 (Win
dows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4577.82 Safari/537.36" "-"
[pid: 45|app: 0|req: 74/464] 10.244.2.1 () {48 vars in 2295 bytes} [Tue Sep 28 14:49:12 2021] GET /api/v2/dashboard/ => generated
 1428 bytes in 292 msecs (HTTP/1.1 200) 15 headers in 670 bytes (1 switches on core 0)
2021-09-28 14:49:13,176 DEBUG    [c7669da896cb451986355a8dd1fc7179] awx.analytics.performance request: <WSGIRequest: GET '/api/v2
/workflow_approvals/?page_size=1&status=pending'>, response_time: 0.169s
[pid: 42|app: 0|req: 143/465] 10.244.2.1 () {48 vars in 2366 bytes} [Tue Sep 28 14:49:13 2021] GET /api/v2/workflow_approvals/?pa
ge_size=1&status=pending => generated 52 bytes in 171 msecs (HTTP/1.1 200) 14 headers in 579 bytes (1 switches on core 0)
10.244.2.1 - - [28/Sep/2021:14:49:13 +0000] "GET /api/v2/workflow_approvals/?page_size=1&status=pending HTTP/1.1" 200 52 "http://
172.17.130.128/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4577.82 Safari/537
.36" "-"
2021-09-28 14:49:13,297 DEBUG    [3b81f8d6480d4418a505b2ebbad5d8ac] awx.analytics.performance request: <WSGIRequest: OPTIONS '/ap
i/v2/organizations/'>, response_time: 0.287s
10.244.2.1 - - [28/Sep/2021:14:49:13 +0000] "OPTIONS /api/v2/organizations/ HTTP/1.1" 200 9656 "http://172.17.130.128/" "Mozilla/
5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4577.82 Safari/537.36" "-"
[pid: 44|app: 0|req: 99/466] 10.244.2.1 () {50 vars in 2342 bytes} [Tue Sep 28 14:49:13 2021] OPTIONS /api/v2/organizations/ => g
enerated 9656 bytes in 289 msecs (HTTP/1.1 200) 14 headers in 581 bytes (1 switches on core 0)
10.244.2.1 - - [28/Sep/2021:14:49:13 +0000] "GET /websocket/ HTTP/1.1" 101 195 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) App
leWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4577.82 Safari/537.36" "-"
2021-09-28 14:49:13,687 DEBUG    [3c768758869b466db9de89cf686ff8ab] awx.analytics.performance request: <WSGIRequest: GET '/api/v2
/auth/'>, response_time: 0.103s
[pid: 42|app: 0|req: 144/467] 10.244.2.1 () {48 vars in 2241 bytes} [Tue Sep 28 14:49:13 2021] GET /api/v2/auth/ => generated 2 b
ytes in 105 msecs (HTTP/1.1 200) 12 headers in 407 bytes (1 switches on core 0)
10.244.2.1 - - [28/Sep/2021:14:49:13 +0000] "GET /api/v2/auth/ HTTP/1.1" 200 2 "http://172.17.130.128/" "Mozilla/5.0 (Windows NT
10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4577.82 Safari/537.36" "-"
2021-09-28 14:49:13,834 DEBUG    [c969ce7935ea48fba27170ae64b50545] awx.analytics.performance request: <WSGIRequest: GET '/api/v2
/auth/'>, response_time: 0.067s
[pid: 45|app: 0|req: 75/468] 10.244.2.1 () {48 vars in 2242 bytes} [Tue Sep 28 14:49:13 2021] GET /api/v2/auth/ => generated 2 by
tes in 68 msecs (HTTP/1.1 200) 12 headers in 407 bytes (1 switches on core 0)
10.244.2.1 - - [28/Sep/2021:14:49:13 +0000] "GET /api/v2/auth/ HTTP/1.1" 200 2 "http://172.17.130.128/" "Mozilla/5.0 (Windows NT
10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4577.82 Safari/537.36" "-"
2021-09-28 14:49:13,840 DEBUG    [74acdd5c833847aea069ec4f3a89043e] awx.analytics.performance request: <WSGIRequest: GET '/api/'>
, response_time: 0.079s
10.244.2.1 - - [28/Sep/2021:14:49:13 +0000] "GET /api/ HTTP/1.1" 200 73885 "http://172.17.130.128/" "Mozilla/5.0 (Windows NT 10.0
; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4577.82 Safari/537.36" "-"
[pid: 42|app: 0|req: 145/469] 10.244.2.1 () {48 vars in 2225 bytes} [Tue Sep 28 14:49:13 2021] GET /api/ => generated 73885 bytes
 in 81 msecs (HTTP/1.1 200) 13 headers in 522 bytes (1 switches on core 0)
2021-09-28 14:49:17,651 DEBUG    [82a22d61a9394bd5b3e6995279c8118d] awx.analytics.performance request: <WSGIRequest: GET '/api/lo
gin/'>, response_time: 0.063s
[pid: 42|app: 0|req: 146/470] 10.244.2.1 () {48 vars in 2237 bytes} [Tue Sep 28 14:49:17 2021] GET /api/login/ => generated 5726
bytes in 65 msecs (HTTP/1.1 200) 10 headers in 483 bytes (1 switches on core 0)
10.244.2.1 - - [28/Sep/2021:14:49:17 +0000] "GET /api/login/ HTTP/1.1" 200 5726 "http://172.17.130.128/" "Mozilla/5.0 (Windows NT
 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4577.82 Safari/537.36" "-"
--> Connection retry
2021-09-28 14:49:17,789 WARNING  [1bb1466ff9334e94a2eaa308e7283c54] awx.api.generics Login failed for user btabac from 10.244.2.1

2021-09-28 14:49:17,797 DEBUG    [1bb1466ff9334e94a2eaa308e7283c54] awx.analytics.performance request: <WSGIRequest: POST '/api/l
ogin/'>, response_time: 0.125s
2021-09-28 14:49:17,798 WARNING  [1bb1466ff9334e94a2eaa308e7283c54] django.request Unauthorized: /api/login/
2021-09-28 14:49:17,798 WARNING  [1bb1466ff9334e94a2eaa308e7283c54] django.request Unauthorized: /api/login/
[pid: 42|app: 0|req: 147/471] 10.244.2.1 () {54 vars in 2388 bytes} [Tue Sep 28 14:49:17 2021] POST /api/login/ => generated 6011
 bytes in 126 msecs (HTTP/1.1 401) 10 headers in 493 bytes (1 switches on core 0)
10.244.2.1 - - [28/Sep/2021:14:49:17 +0000] "POST /api/login/ HTTP/1.1" 401 6011 "http://172.17.130.128/" "Mozilla/5.0 (Windows N
T 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4577.82 Safari/537.36" "-"
```

**Comment by atsigr** [comment]
Same problem. I can confirm. We use ldap from rhel idm IPA.

**Comment by CWollinger** [comment]
Same problem with AWX 19.3.0. Issue fixed in AWX 19.4.0. Releated issue: https://github.com/ansible/awx/issues/10883

**Comment by shanemcd** [comment]
As stated in the comment above, this should be fixed with 19.4.0.

**Comment by ktibi** [comment]
@shanemcd Not fixed in 19.4.0 for me.

**Comment by heizac** [comment]
I was having the same problem. Upgrading to 19.4.0 fixed the issue.
